### ISSUE TITLE
Problems with kill/yank in tests

### ISSUE THREAD

**Issue by nispio** [magnars/multiple-cursors.el#296]
I am trying to write feature tests for cask/ecukes that involve kill/yank with multiple cursors.  They all work when I run them on my local machine, but when they run on Travis CI they fail in strange ways.  Here is one example of a failed scenario that seems pretty straightforward to me:

```
  Scenario: Separate kills
    When I insert:
      """
      aaa
      bbb
      ccc
      """
    And I go to the front of the word "aaa"
    And I press "C->"
    And I press "C->"
    And I press "C-SPC"
    And I press "C-e"
    And I press "C-w"
    And I type "_"
    And I press "C-y"
    Then I should see:
      """
      _aaa
      _bbb
      _ccc
      """
```

This succeeds locally, but when it runs on Travis CI, I get the following error:

```
      Expected
      _aaa
      _bbb
      _ccc
      to be part of:
      _aaa
      bbb_
      ccc_
```

Does anyone have any insights into why the behavior is different?

This is related to PR #295 and you can see the Travis logs there.

**Comment by nispio** [comment]
@magnars, @jtamagnan, any ideas on this?  I would love to get this merged but I don't even know how to continue debugging the test without submitting lots of tiny commits to the PR branch.  Am I going to annoy everyone if I keep doing that?

**Comment by jtamagnan** [comment]
Yeah that seems like the only way I can think of or trying to recreate the travis environment as per this stack overflow post https://stackoverflow.com/questions/21053657/how-to-run-travis-ci-locally and seeing if the same issues happen

**Comment by nispio** [comment]
The main thing I see that differentiates this test from existing tests that do similar things is the use of `C-w` and `C-y`.  None of the existing tests use those to kill multiple regions.  Is there perhaps some multiple-cursors-aware version of kill and yank that I need to bind those keys to?
